### ISSUE TITLE
[Test] Investigate console warnings in get_local_stats.test.ts

### ISSUE THREAD

**Issue by ananzh** [opensearch-project/OpenSearch-Dashboards#556]
**Describe the issue**

Currently, `get_local_stats.test.ts` prints out console warnings when run the unit test. To make a clean unit test, we want to explore the reasons for these errors/warnings and decide whether we could fix or remove them. 

```
  console.warn
    No OpenSearchDashboards stats returned from usage collectors

      60 | ) {
      61 |   if (!response) {
    > 62 |     logger.warn('No OpenSearchDashboards stats returned from usage collectors');
         |            ^
      63 |     return;
      64 |   }
      65 |   const {

      at handleOpenSearchDashboardsStats (src/plugins/telemetry/server/telemetry_collection/get_opensearch_dashboards.ts:62:12)
      at handleLocalStats (src/plugins/telemetry/server/telemetry_collection/get_local_stats.ts:73:30)
      at Object.it (src/plugins/telemetry/server/telemetry_collection/get_local_stats.test.ts:199:22)

  console.warn
    No OpenSearchDashboards stats returned from usage collectors

      60 | ) {
      61 |   if (!response) {
    > 62 |     logger.warn('No OpenSearchDashboards stats returned from usage collectors');
         |            ^
      63 |     return;
      64 |   }
      65 |   const {

      at handleOpenSearchDashboardsStats (src/plugins/telemetry/server/telemetry_collection/get_opensearch_dashboards.ts:62:12)
      at handleLocalStats (src/plugins/telemetry/server/telemetry_collection/get_local_stats.ts:73:30)
      at Object.it (src/plugins/telemetry/server/telemetry_collection/get_local_stats.test.ts:216:22)


```

**To Reproduce**
Run:`yarn test:jest {path to get_local_stats.test.ts}`

**Tasks**
    
I think the logger warning is being called as expected, indicating that the code is handling some specific case. The fact that the logger is outputting a warning is not a problem in itself, as long as it's the expected behavior in this specific test case. But it's essential to understand the context of the test and the purpose of the warning message. If the test is designed to check the handling of missing OpenSearchDashboards stats, then this warning message is expected and normal.

To sum up, this is a good research task to help us understand the testing context and make sure there is no issue to see logger warning. To close the issue, pls answer:

- What does this test test about? What is the purpose of the warning message?
- Is it expected to see logger warning? If yes, close the issue.
- If no, please comment in the issue and raise a PR.

**Comment by zengyan-amazon** [comment]
Closing due to extended inactivity (4+ years with no comments). The test suite has been significantly reworked since this was filed, and the specific test file may no longer exist or have these warnings. If this is still relevant, please feel free to reopen or create a new issue. Thank you!
